src/init.php

```diff
@@ -188,6 +188,10 @@ public function load_frontend_scripts_conditionally_head() {
 		 * @return string output block
 		 */
 		public function load_frontend_scripts_conditionally( $block_content, $block ) {
+			if ( $block_content === null ) {
+				return $block_content;
+			}
+
```

Comment:
Same as above, please just handle when block_content is null because this will skip some processes